test/Core.Test/Services/UserServiceTests.cs

```diff
@@ -175,6 +173,11 @@ public async Task Needs2FABecauseNewDeviceAsync_ReturnsTrue(SutProvider<UserServ
                             new Device { Identifier = deviceIdInRepo }
                        }));
 
+            sutProvider.GetDependency<Settings.GlobalSettings>().TwoFactorAuth = new Settings.GlobalSettings.TwoFactorAuthSettings
+            {
+                EmailOnNewDeviceLogin = true
+            };
+
```

Comment:
We're trying to get away from using our default global settings injector.

Once you make the changes to inject and interface you should be able to:
```
sutProvider.GetDependency<IGlobalSettings>().TwoFactorAuth.EmailOnNewDevice.Returns(true)
```